sonarts-core/docs/rules/default-clauses-should-be-last.md

```diff
@@ -0,0 +1,48 @@
+# default-clauses-should-be-last
```

Comment:
@andrea-guarino-sonarsource think of something more minimalistic for rule key (yeah, I understand that it's a big pain to rename everything :), that's why it's better to discuss name with somebody if you are not sure)
How about `prefer-default-last`?